src/linear_algebra/basis.lean

```diff
@@ -341,6 +341,37 @@ lemma is_basis.smul_eq_zero [no_zero_divisors R] {c : R} {x : M} :
 
 end
 
+lemma is_basis.smul_of_is_unit {v : ι → M} (hv : is_basis R v)
+  {w : ι → R} (hw : ∀ i : ι, is_unit (w i)) :
+  is_basis R (λ i, w i • v i) :=
```

Comment:
You can actually write this:
```suggestion
  is_basis R (w • v) :=
```

Reply:
It's definitely cleaner and it didn't introduce too many errors